tools/dgeni/src/templates/api/base.template.html

```diff
@@ -1,5 +1,5 @@
 {% block header %}
-<h1>{$ doc.name $}</h1>
+{$'<h3>' if child else '<h1>'$}{$ doc.name $}{$'</h3>' if child else '</h1>'$}
```

Comment:
@xelaint Does `<h3>` and `<h4>` seem good to you for API docs in child mode (like in the API tab in component doc pages).

Reply:
`<h3>` and `<h4>` are good